### ISSUE TITLE
Error: "method setContentPermissions taking 3 parameters" when trying to lock page

### ISSUE THREAD

**Issue by Netzeband** [kovetskiy/mark#17]
Hello

I'm using Confluence 7.2.0 and when I upload a markdown file together with the request to lock the page (option `-k`) I get the following error code:

```
2020-02-20 06:24:30 [FATAL] 'true' response expected, but 'map[error:map[code:-32602 data:<nil> message:Request parameter types did not match method parameter types (method setContentPermissions taking 3 parameters)] id:<nil> jsonrpc:2.0]' encountered
```

Just uploading the page works well. In the `api.go` file I can see, that the function `RestrictPageUpdatesServer` passes indeed 3 arguments to the API call, so I don't understand why Confluence is complaining here. Maybe the API changed?

**Comment by elgreco247** [comment]
The signature of the json-rpc call in `RestrictPageUpdatesServer` wasn't right: #45

**Comment by kovetskiy** [comment]
Closing because fixed by #45 
Thanks to @elgreco247
